apps/guardian-ui/src/languages/de.json

```diff
@@ -17,7 +17,13 @@
     "you": "Du",
     "close": "Schließen",
     "save": "Speichern",
-    "continue": "Fortsetzen"
+    "continue": "Fortsetzen",
+    "submit": "Einreichen"
+  },
+  "notConfigured": {
+    "title": "Nicht konfiguriert",
+    "description": "Bitte geben Sie die CONFIG_API_URL des Fedimint-Wächter-Servers ein, um fortzufahren.",
+    "urlPlaceholder": "wss://fedimintd.my-awesome-domain.com:6000"
```

Comment:
nit: this doesn't have to be a locale imo since they're all the same

Reply:
removed

Reply:
The German translation is hilarious :laughing: WÄCHTERSERVER!!!